spek-junit-platform-engine/src/test/kotlin/org/jetbrains/spek/engine/LifecycleListenerOrderTest.kt

```diff
@@ -72,12 +72,12 @@ class LifecycleListenerOrderTest: AbstractSpekTestEngineTest() {
         executeTestsForClass(TestSpek::class)
         // afterXXX fixtures are executed from bottom to top
         val expected = """
-        1
         3
         2
         1
         3
         2
+        1
```

Comment:
To be clear, this is kinda breaking change (and below in the same file), we need to try estimate whether it can break some users specs or not